.github/workflows/metadata_diff.yml

```diff
@@ -4,7 +4,6 @@ on:
     inputs:
       reference_binary:
         description: The link to the reference binary, which should be the previous release.
-        default: https://github.com/Manta-Network/Manta/releases/download/v4.0.5/manta
```

Comment:
Do we not need this line?

Reply:
it's just a pain to maintain and this job is always run manually, so no